common/test/acceptance/pages/lms/discussion.py

```diff
@@ -69,6 +70,7 @@ def get_group_visibility_label(self):
 
     def get_response_total_text(self):
         """Returns the response count text, or None if not present"""
+        self.wait_for_ajax()
```

Comment:
This data is also populated by ajax callbacks but we can't wait for it specifically, as sometimes it is correctly not present.